content-security-policy/generic/generic-0_1-img-src.html.sub.headers

```diff
@@ -0,0 +1,6 @@
+Expires: Mon, 26 Jul 1997 05:00:00 GMT
+Cache-Control: no-store, no-cache, must-revalidate
+Cache-Control: post-check=0, pre-check=0, false
+Pragma: no-cache
+Set-Cookie: generic-0_1-img-src={{$id:uuid()}}; Path=/content-security-policy/generic/
+Content-Security-Policy: default-src 'self' 'unsafe-inline'; report-uri  ../support/report.py?op=put&reportID={{$id}}
```

Comment:
because of this and the way the test is written, you are implicitly testing the cascade to script-src too and also why you need to add 'unsafe-inline' to default-src. Maybe make it a separate script-src directive?